src/handlers/serenity.rs

```diff
@@ -217,6 +217,14 @@ impl SerenityHandler {
                                 .required(true)
                                 .min_int_value(1)
                         })
+                        .create_option(|option| {
+                            option
+                                .name("until")
+                                .description("Add a subsequent track position to remove a range of tracks")
```

Comment:
We aren't describing options in declarative style, maybe something "**Upper range track position to remove multiple tracks**" is more suitable.

Reply:
I like it!